ecaudit/src/main/java/com/ericsson/bss/cassandra/ecaudit/metrics/AuditMetrics.java

```diff
@@ -78,14 +79,11 @@ public void logAuditRequest(long time, TimeUnit timeUnit)
 
     /**
      * Copied from org.apache.cassandra.metrics.DefaultNameFactory but with tailored group name and slightly reduced.
+     *
      * @return the name of the mBean.
      */
     private static String createMBeanName(String name)
     {
-        return new StringBuilder()
-               .append(GROUP_NAME)
-               .append(":type=").append(METRIC_TYPE)
-               .append(",name=").append(name)
-               .toString();
+        return GROUP_NAME + ":type=" + METRIC_TYPE + ",name=" + name;
```

Comment:
I'd suggest we create another constant like MBEAN_BASE that is everything up to name.